style.css

```diff
@@ -130,3 +130,467 @@ p {
 .menu-bar {
   display: none;
 }
+
+/* work section */
+.work {
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.work-section {
+  display: grid;
+  grid-template-columns: 100%;
+}
+
+.work-1 {
+  display: flex;
+  flex-direction: column;
+}
+
+.work1-visual {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.work1-visual > h1 {
+  font-family: "Crete Round", serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 52px;
+  letter-spacing: 0;
+  align-items: center;
+  color: #1c1a19;
+  margin-left: 10px;
+  margin-top: 80px;
+}
+
+.line {
+  margin-top: -90px;
+  width: 327px;
+  height: 0;
+  border: thin solid black;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
```

Comment:
- [ ] kindly Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels it best practices.
- [ ] your Ui display
![Screenshot 2023-06-02 052156](https://github.com/sulaimanahmadi/Portfolio/assets/92458236/81c43080-8532-477b-880f-fc05c9829a6a)